lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -38,6 +38,12 @@
 DEFAULT_DNS_IPV6_ADDR = "2001:4860:4860:0:0:0:0:8888"
 DEFAULT_P_CSCF_IPV4_ADDR = "172.27.23.150"
 DEFAULT_P_CSCF_IPV6_ADDR = "2a12:577:9941:f99c:0002:0001:c731:f114"
+DEFAULT_NGAP_S_NSSAI_SST = "1"
+DEFAULT_NGAP_S_NSSAI_SD = "0xffffff"
+DEFAULT_NGAP_AMF_NAME = "AMF_1"
+DEFAULT_NGAP_AMF_REGION_ID = "1"
+DEFAULT_NGAP_SET_ID = "1"
+DEFAULT_NGAP_AMF_POINTER = "1"
```

Comment:
this value for amf pointer is 1 but on the gateway.mconfig is set to 3, let's set them both to the expected default value. cc @ulaskozat

Reply:
Comments addressed. The default value of pointer should be 0.